java/datastructures/lib/src/test/java/datastructures/hashmaptest/hashMapTest.java

```diff
@@ -0,0 +1,54 @@
+package datastructures.hashmap;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import java.util.*;
+import static org.junit.jupiter.api.Assertions.*;
+
+public class hashMapTest {
+  private HashMap<String, Integer> hashMap;
+
+  @BeforeEach
+  public void setUp() {
+    hashMap = new HashMap<>(5);
+  }
+
+  @Test
+  public void setGetTest() {
+    hashMap.set("keyOne", 1);
+    hashMap.set("keyTwo", 2);
+
+    assertEquals(1, hashMap.get("keyOne"));
+    assertEquals(2, hashMap.get("keyTwo"));
+  }
+
+  @Test
+  public void doubleKeysTest() {
+    hashMap.set("keyOne", 1);
+    hashMap.set("keyTwo", 2);
```

Comment:
I think you meant `set("keyOne", 2)` here; with that correction, the test passes.